output/report.go

```diff
@@ -44,7 +44,9 @@ func GetReporter(names []string) (Reporter, error) {
 	for _, name := range names {
 		switch name {
 		case "list":
-			mux = append(mux, List)
+			mux = append(mux, &List{Compact: false})
+		case "compact":
+			mux = append(mux, &List{Compact: true})
```

Comment:
I think keeping things simple is more important. But for the scope, it doesn't matter much, and I will split into two distinct reporters.